geoportal/demo_geoportal/locale/de/LC_MESSAGES/demo_geoportal-client.po

```diff
@@ -423,10 +425,10 @@ msgid "osm"
 msgstr "OSM"
 
 msgid "osm_id"
-msgstr "OSM zeitbezogen"
+msgstr ""
```

Comment:
?

Reply:
I removed the translation, because it made no sense (and furthermore, the same translation was used for lots of different elements)

Reply:
Just the latest missing :-)